src/compiler/reviews.md

```diff
@@ -64,10 +66,8 @@ not it should be part of a [rollup] with one of the following:
       rollup).
     - Has a high chance of failure.
     - Is otherwise dangerous to rollup.
-
-> **Note**:\
-> `@bors rollup` is equivalent to `@bors rollup=always`\
-> `@bors rollup-` is equivalent to `@bors rollup=never`
+- `rollup`: this is equivalent to `rollup=always`
+- `rollup-`: this is equivalent to `rollup=never`
```

Comment:
```suggestion
- `rollup-`: this is equivalent to `rollup=maybe`
```

This is wrong, it's equivalent to `maybe`. I know the original is incorrect but let's take the opportunity to fix it :)

Reply:
oops, TIL!